aptos-move/framework/aptos-token/sources/property_map.move

```diff
@@ -83,10 +100,36 @@ module aptos_token::property_map {
     ): (String, PropertyValue) {
         let found = contains_key(list, key);
         assert!(found, EPROPERTY_NOT_EXIST);
-
+        let (found, ind) = vector::index_of(&mut list.keys, key);
+        assert!(found, EPROPERTY_NOT_EXIST);
+        vector::remove(&mut list.keys, ind);
         simple_map::remove(&mut list.map, key)
     }
 
+    /// update the property in the existing property map
+    public fun update_property_map(
```

Comment:
Same question here. What's the use case?

Reply:
same as above. Introduced to support the changes in token.move

Reply:
also mentioned in the PR description on rationals behind this token change.